src/main/java/net/dries007/tfc/objects/recipes/QuernRecipeManager.java

```diff
@@ -0,0 +1,134 @@
+/*
```

Comment:
Remove this

Reply:
I don't know why the deletes didn't work